doc/ale-graphql.txt

```diff
@@ -2,6 +2,9 @@
 ALE GraphQL Integration                                   *ale-graphql-options*
 
 
+===============================================================================
+eslint                                                     *ale-graphql-eslint*
```

Comment:
You should document that the JavaScript options are also used for GraphQL here. See `:help ale-typescript-eslint`.

Reply:
Sorry, I'm not familiar at all with writing vim docs - could you point me to an analogous location I can copy?

Reply:
Yes, that help tag points to it. See `ale-typescript.txt`.

Reply:
`*foo*` creates a help tag for `foo`, and `|foo|` references a help tag.